src/index.ts

```diff
@@ -22,7 +22,7 @@ export {
   getRouteContext,
```

Comment:
Looks like you missed a case in the `MemoryRouter`since `MemoryRouterProps` is exported in the index.tsx file.